.circleci/config.yml

```diff
@@ -162,95 +164,94 @@ jobs:
-  #           branches:
-  #             only: release
-  #    - terraform-init-and-apply-to-staging:
-  #        requires:
-  #          - assume-role-staging
-  #        filters:
-  #          branches:
-  #            only: release
-  #    - deploy-to-staging:
-  #        context: api-nuget-token-context
-  #        requires:
-  #          - terraform-init-and-apply-to-staging
-  #        filters:
-  #          branches:
-  #            only: release
-  #    - permit-production-terraform-release:
-  #        type: approval
-  #        requires:
-  #          - deploy-to-staging
-  #        filters:
-  #          branches:
-  #            only: release
-  #    - assume-role-production:
-  #        context: api-assume-role-housing-production-context
-  #        requires:
-  #            - permit-production-terraform-release
-  #        filters:
-  #           branches:
-  #             only: release
-  #    - terraform-init-and-apply-to-production:
-  #        requires:
-  #          - assume-role-production
-  #        filters:
-  #          branches:
-  #            only: release
-  #    - permit-production-release:
-  #        type: approval
-  #        requires:
-  #          - deploy-to-staging
-  #          - terraform-init-and-apply-to-production
-  #        filters:
-  #          branches:
-  #            only: release
-  #    - deploy-to-production:
-  #        context: api-nuget-token-context
-  #        requires:
-  #          - permit-production-release
-  #          - terraform-init-and-apply-to-production
-  #        filters:
-  #          branches:
-  #            only: release
+          filters:
+            branches:
+              only: master
+      - assume-role-staging:
+          context:
+            - api-assume-role-staging-context
+            - SonarCloud
+          requires:
+            - build-and-test
+          filters:
+            branches:
+              only: master
+      - terraform-init-and-apply-to-staging:
+          requires:
+            - assume-role-staging
+          filters:
+            branches:
+              only: master
+      - migrate-database-staging:
+          requires:
+            - terraform-init-and-apply-to-staging
+          filters:
+            branches:
+              only: master
+      - deploy-to-staging:
+          requires:
+            - migrate-database-staging
+          filters:
+            branches:
+              only: master
+      - permit-production-terraform-release:
+          type: approval
+          requires:
+            - deploy-to-staging
+      - assume-role-production:
+          context: api-assume-role-housing-production-context
+          requires:
+            - permit-production-terraform-release
+          filters:
+            branches:
+              only: master
+      - migrate-database-production:
+          requires:
+            - terraform-init-and-apply-to-production
+          filters:
+            branches:
+              only: master
```

Comment:
Same as the comment about migration on staging.
The production migration job doesn't exist.